### ISSUE TITLE
64 bit version of meshlab suggests I use the 64 bit version of meshlab.

### ISSUE THREAD

**Issue by fake-name** [cnr-isti-vclab/meshlab#600]
I'm trying to do some filtering on **very** large meshes, and have been hitting memory errors.

Right now, Any filter failure throws a error [here](https://github.com/cnr-isti-vclab/meshlab/blob/d0fa7a31f0e6c4796e5d972a7e9ebb6e950a1b55/src/meshlab/mainwindow_RunTime.cpp#L1622):

```
        qApp->restoreOverrideCursor();
        QMessageBox::warning(this, tr("Filter Failure"), QString("Operating system "
        "was not able to allocate the requested memory.<br><b>Failure of filter "
        "<font color=red>: '%1'</font><br>We warmly suggest you to try a 64-bit "
        "version of MeshLab.<br>").arg(action->text())+bdall.what()); // text
        MainWindow::globalStatusBar()->showMessage("Filter failed...",2000);
```

I'm on a 64 bit build. Perhaps this should report something like "Either simplify your model, or try on a machine with more memory" if it's a 64 bit build.

**Comment by alemuntoni** [comment]
Solved in #635, thanks @fake-name to reporting it!

**Comment by fake-name** [comment]
Thanks! FWIW this was a very silly issue.
